product_landing_page/index.css

```diff
@@ -158,4 +158,11 @@ body {
   .get-started-section__title {
     text-align: center;
   }
+
+  .container__item--video {
```

Comment:
`.container__item--video` nie jest częścią tego ficzera, więc lepiej usunąć to z tego PRa :)

Reply:
sorki, to już do kolejnego PR mialo byc :D